frontend/plan/src/components/search/SearchBar.js

```diff
@@ -71,6 +75,84 @@ function SearchBar({
+                                startSearch={conditionalStartSearch}
+                                schoolReq={schoolReq}
+                                filterData={filterData}
+                                addSchoolReq={addSchoolReq}
+                                remSchoolReq={remSchoolReq}
+                                isDisabled={isLoading}
+                            />
+                        </DropdownButton>
+                        <DropdownButton title="Difficulty" filterData={filterData.difficulty} defaultFilter={defaultFilters.filterData.difficulty} clearFilter={clearFilterSearch("difficulty")} isDisabled={isLoading}>
+                            <RangeFilter
+                                minRange={0}
+                                maxRange={4}
+                                step={0.25}
+                                filterData={filterData}
+                                updateRangeFilter={updateRangeFilter("difficulty")}
+                                startSearch={conditionalStartSearch}
+                                rangeProperty="difficulty"
+                                isDisabled={isLoading}
+                            />
+                        </DropdownButton>
+                        <DropdownButton title="Course Quality" filterData={filterData.course_quality} defaultFilter={defaultFilters.filterData.course_quality} clearFilter={clearFilterSearch("course_quality")} isDisabled={isLoading}>
+                            <RangeFilter
+                                minRange={0}
+                                maxRange={4}
+                                step={0.25}
+                                filterData={filterData}
+                                updateRangeFilter={updateRangeFilter("course_quality")}
+                                startSearch={conditionalStartSearch}
+                                rangeProperty="course_quality"
+                                isDisabled={isLoading}
+                            />
+                        </DropdownButton>
+                        <DropdownButton title="Instructor Quality" filterData={filterData.instructor_quality} defaultFilter={defaultFilters.filterData.instructor_quality} clearFilter={clearFilterSearch("instructor_quality")} isDisabled={isLoading}>
+                            <RangeFilter
+                                minRange={0}
+                                maxRange={4}
+                                step={0.25}
+                                filterData={filterData}
+                                updateRangeFilter={updateRangeFilter("instructor_quality")}
+                                startSearch={conditionalStartSearch}
+                                rangeProperty="instructor_quality"
+                                isDisabled={isLoading}
+                            />
+                        </DropdownButton>
+                    </div>
+                )}
+            </div>
+        );
+    }
+
```

Comment:
there's some functionality difference as well, since the filters only show up on mobile on button press -- also, the button for filtering is different, and the sizing and spacing of the elements is so different that I feel like it would be more confusing using CSS.

but I did abstract the repeated code (dropdowns) so that it's not written twice